LayoutTests/imported/w3c/web-platform-tests/import-maps/no-referencing-script-integrity.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+let log = [];
+</script>
+<script type="importmap">
+{
+  "integrity": {
+    "./resources/log.js?pipe=sub&name=NoReferencingScriptInvalidCheck": "sha384-Li9vy3DqF8tnTXuiaAJuML3ky+er10rcgNR/VqsVpcw+ThHmYcwiB1pbOxEbzJr7"
+  }
+}
+</script>
+<script>
+let promiseResolve;
```

Comment:
As above... gutting the promise seems unnecessary?

Reply:
Same. This test structure is there to guarantee that the import happens as part of an HTML-based event

Reply:
As above... just for improved legibility with the same functionality?